test/compose/history/rc

```diff
@@ -0,0 +1,7 @@
+# Make our expansion have a predictable timestamp
+hook global ClientClose .* %{
+    evaluate-commands %sh{
```

Comment:
We dont have any kakoune commands we want to evaluate from this output, so `nop` would be better.